packages/block-editor/src/components/block-settings-menu/index.js

```diff
@@ -134,10 +134,10 @@ export function BlockSettingsMenu( { clientIds } ) {
 											onToggle={ onClose }
 										/>
 									) }
-									<__experimentalBlockSettingsMenuPluginsExtension.Slot
-										fillProps={ { clientIds, onClose } }
-									/>
 								</MenuGroup>
+								<BlockSettingsMenuControls.Slot
```

Comment:
By making the slot a new `MenuGroup` we are introducing a visual difference (notice the separator before the reusable blocks item in the menu). 

Before | After
--- | ---
![2002-blocksettingsmenu-before](https://user-images.githubusercontent.com/583546/74729343-9805eb80-5244-11ea-8608-95da230605c8.png) |  ![2002-blocksettingsmenu-after](https://user-images.githubusercontent.com/583546/74729351-9c320900-5244-11ea-99ec-03f685592f30.png)

I think this is fine, especially because that area is _variable_ (third-parties can hook into and add new contents). Raised in case this needed a design :+1:

Reply:
FWIW, it doesn't change the keyboard interaction.

Reply:
Yes, this seems fine.

Reply:
For completeness, this is how it looks with plugins:

![2002-blocksettingsmenu-after-with-plugin](https://user-images.githubusercontent.com/583546/74734352-ad801300-524e-11ea-88d8-fd41c229bb77.png)

Plugins are added before the core utils (reusable blocks, groups), as it works today.

Reply:
> Plugins are added before the core utils (reusable blocks, groups), as it works today.

This is a bit unfortunate. I think before we had a slot in another slot to make it work. Should we bring it back even though you (and me) liked the simplification? :)

Reply:
I meant that it works the same as it did! In `master`, the plugins are also loaded before reusable and group items. If anything, this PR visually clarifies where plugins are added due to the new separator.